guaymas_data_analysis/utils/data_utils.py

```diff
@@ -52,7 +52,7 @@ def convert_oceanographic_measurements(df: pd.DataFrame, vehicle="sentry"):
     # calculate practical salinity
     df.loc[:, "practical_salinity"] = df.apply(
         lambda x: gsw.SP_from_C(
-            C=x["ctd_cond"],
+            C=x["ctd_cond"]*10,
```

Comment:
Would be good to add a comment here for why the multiplication by 10 is necessary

Reply:
Good catch -- this is just a unit conversion from S/m to mS/cm as per the GSW library input guidelines. Added a comment!